WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -86,7 +87,38 @@ final class TotalsViewModel: ObservableObject {
         startSyncingOrder(with: cartItems, allItems: allItems)
     }
 
+    private func areOrderAndCartDifferent(cartItems: [CartItem]) -> Bool {
+        // check if order has same items as cart does
+        if let order {
+            // first we get list of all products 1 by 1
+            var cleanOrderItems: [POSOrderItem] = order.items.flatMap {
+                Array(repeating: $0, count: $0.quantity.intValue)
+            }
+            var cleanCartItems: [POSItem] = cartItems.flatMap {
+                Array(repeating: $0.item, count: $0.quantity)
+            }
+            if cleanOrderItems.count == cleanCartItems.count {
+                // sort items by productIDs to have them in same order for comparison
+                cleanOrderItems.sort { $0.productID < $1.productID }
+                cleanCartItems.sort { $0.productID < $1.productID }
+                // check if all the items are same, prices included
+                for (index, cartItem) in cleanCartItems.enumerated() {
```

Comment:
The `cartItem` here is of type `POSItem`, is this expected? I just realized because `CartItem` or `POSCartItem` doesn't have a price property that can be accessed directly, and needs to be access through it's internal `product` property